lighty-modules/lighty-openflow-sb/src/main/java/io/lighty/modules/southbound/openflow/impl/OpenflowSouthboundPlugin.java

```diff
@@ -211,10 +215,18 @@ protected boolean initProcedure() {
     }
 
     @Override
+    @SuppressWarnings("checkstyle:illegalCatch")
     protected boolean stopProcedure() {
         destroy(this.packetListenerNotificationRegistration);
         destroy(this.flowCapableTopologyProvider);
         destroy(this.operationProcessor);
+        if (this.listenerRegistrationHelper != null) {
+            try {
+                listenerRegistrationHelper.close();
+            } catch (final Exception e) {
+                LOG.warn("Exception was thrown during closing listenerRegistrationHelper", e);
```

Comment:
At the end of this method you should return `false` if anything failed.

Reply:
done